payments-core/src/main/java/com/stripe/android/model/PaymentIntent.kt

```diff
@@ -226,9 +226,8 @@ constructor(
     }
 
     private fun isLpmLevelSetupFutureUsageSet(code: PaymentMethodCode): Boolean {
-        return paymentMethodOptionsJsonString?.let { json ->
-            val pmOptions = JSONObject(json).optJSONObject(code)
-            pmOptions?.has("setup_future_usage") ?: false
+        return paymentMethodOptionsMap?.let { options ->
```

Comment:
Can someone double check the logic here? We are now checking if the `setup_future_usage` is set with a value, rather than null.